mobileversion.html

```diff
@@ -0,0 +1,231 @@
+     <title>Mobile Version Website</title>
+  </head>
+    <body>
+       <header class="header">
+        <nav>
+     <a href="#">My Logo</a>
+     <a href="#"><i class="fa fa-bars"></i></a>
+     </nav>
+        </header>
+             <section>
+                <div class="flex-container">
+                    <div class="item5">
+            </div>
+            <div class="item6">
+              <div>
+                <h1 class="mariane">I'm Mariane<br>
+                  Glad to see<br>
+                 you!<br>
+               </h1>
+              </div>
+                <div>
+                <p class="text">
+                  I’m a software developer! I can help you<br>
+                   build a product , feature or website Look <br>
+                   through some of my work and<br>
+                    experience! If you like what you see and <br>
+                    have a project you need coded, don’t<br>
+                     hestiate to contact me.
+                </p>
+              </div>
+              <div >
+                <h2 class="cap">LET’S CONNECT</h2>
+                <ul class="lass">
+                  <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+                  <li><a href="#"><i class='fab fa-linkedin-in'></i></a></li>
+                  <li><a href="#"><i class='fab fa-medium-m'></i></a></li>
+                  <li><a href="#"><i class='fab fa-github'></i></a></li> 
+                  <li><a href="#"><i class='fab fa-angellist'></i></a></li> 
+                </ul>
+              </div>
+              </div>
+           </div>
+        </section>
+        <section>
+       <div class="container">
+         <div class="item1">
+         <img src="https://cdn.dribbble.com/users/472534/screenshots/6591791/blog_drbl.png" alt="snapshoot">
+         <div class="ton">
+            <h1 class="tonic">Tonic</h1>
+         </div>
+         <div class="sec">
+            <p class="mainp">
+            <h5 class="canopy">CANOPY</h5>
+            <p class="mid">&#8226;</p>
+            <p class="back">Back End Dev</p>
+            <p class="mid">&#8226;</p>
+            <p class="year">2015</p>
+            </p>
+         </div>
+         <div class="day">
+            <p class="daily">
+               A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+         </div>
+         <div class="maincon">
+            <ul>
+         <li class="container2"><a href="#">html</a></li>
+         <li class="container3"><a href="#">css</a></li>
+         <li class="container4"><a href="#">javaScript</a></li>
+         </ul>
+      </div>
+      <div class="see">
+         <a href="#" class="pro">See Project</a>
+      </div>
+      </div>
+      </div>
+      <div class="container">
+        <div class="item1">
+            <img src="C:\Users\Harrison\Downloads\Snapshoot Portfolio (3).png" alt="multiimage2">
```

Comment:
[OPTIONAL] Please fix the path to this image. Currently it does not display.
<img width="1316" alt="Screenshot 2022-10-29 at 15 18 00" src="https://user-images.githubusercontent.com/48546992/198831120-ddc62122-fbdb-4e6f-a769-97a2a3a6dcca.png">